mysql-test/include/partition_with_compressed_columns.inc

```diff
@@ -0,0 +1,30 @@
+# ==== Purpose ====
```

Comment:
Add percona_ prefix to the test file

Reply:
@laurynas-biveinis ready for next review